src/test/java/com/puppycrawl/tools/checkstyle/filters/XpathFilterElementTest.java

```diff
@@ -223,7 +248,7 @@ public void testNonMatchingFileNameModuleIdAndCheck() throws Exception {
     public void testNullModuleIdAndNonMatchingChecks() throws Exception {
         final String xpath = "NON_MATCHING_QUERY";
         final XpathFilterElement filter = new XpathFilterElement(
-                "InputXpathFilterElementSuppressByXpath", "NonMatchingRegexp", null, null, xpath);
+                (String) null, null, null, null, xpath);
```

Comment:
Please move such updates to separate commit of the same PR, as it is not related to issue